funlet-echo/.env

```diff
@@ -1,3 +1,8 @@
+# description: The default TwiML to echo back (can be overridden by the Twiml URL parameter)
+# format: text
+# link: https://www.twilio.com/docs/voice/twiml
+FUNLET_ECHO_TWIML="<Response><Say>Hello, world!</Say></Response>"
```

Comment:
is there a reason we have to wrap this in quotes as opposed to all other examples?

Reply:
My editor threw a parse error when encountering `=<` here, but after removing the quotes this still seems to pass the parsing parts of `npm test`.